docs/onboarding/basics/7-polkadot.md

```diff
@@ -0,0 +1,41 @@
+# Polkadot
+
+Polkadot is a blockchain of blockchains. Instead of executing smart contracts directly, it allows other chains to connect and verifies the integrity of the connected chains. We refer to this as 'shared security'.
+
+- [What is Polkadot? A Brief Introduction](https://polkadot.network/blog/what-is-polkadot-a-brief-introduction/)
+
+[![What is Polkadot? - Video](https://img.youtube.com/vi/BQ60bTU1bPg/maxresdefault.jpg)](https://youtu.be/BQ60bTU1bPg)
+
+Navigate to the [wiki](https://wiki.polkadot.network/docs/learn-accounts), and make sure to at least cover all the topics listed under `basics`. These include `Accounts`, `Tokens and Assets`, `Components`, and `Staking`.
+
+## Composable and Picasso
+
+Parachains are a part of our product. We use these as bridging hubs for our `XCVM` network, as well as an execution layer for DeFi protocols. Initially, we won two slots, the Picasso (Kusama) slot #13: leasing November 30, 2021 - November 2, 2022, and the Composable (Polkadot) slot #8: leasing March 11, 2022 - January 16, 2024.
+
+Some products that we are building and deploying on Picasso and Composable include:
+
+- [Pablo](https://dali.devnets.composablefinance.ninja/products/pablo-overview.html)
+- [Centauri](https://dali.devnets.composablefinance.ninja/products/centauri-overview.html)
+- [Composable's-XCVM](https://dali.devnets.composablefinance.ninja/products/cross-chain-virtual-machine.html)
+- [CosmwasmVM](https://dali.devnets.composablefinance.ninja/products/cross-chain-virtual-machine/writing-smart-contracts-with-cosmwasm.html)
+
+
+
+> **Note**
+> While writing this document, links to these products were not available yet. Please ask your lead if they already are, and if so, file a PR with the documentation of each.
+
+In conjunction with our main projects, Composable Labs is incubating more projects to deploy on our chains.
+
+## Acala
+
+A parachain focussed on DeFi and stablecoins, and one of our partners in the DeFi space. Read through their [wiki](https://wiki.acala.network/) on the different products as well as their EVM+
+
+## Moonbeam
+
+Another major parachain, focussing on being an Ethereum compatible execution layer. Many protocols jumped over to moonbeam because they did not have to rewrite their contracts.
```

Comment:
```suggestion
Another major parachain, focusing on being an Ethereum compatible execution layer. Many protocols jumped over to moonbeam because they did not have to rewrite their contracts.
```